networkx/algorithms/centrality/flow_matrix.py

```diff
@@ -16,9 +16,9 @@ def flow_matrix_row(G, weight='weight', dtype=float, solver='lu'):
     C = solvername[solver](L, dtype=dtype) # initialize solver
     w = C.w # w is the Laplacian matrix width
     # row-by-row flow matrix
-    for u,v,d in G.edges(data=True):
+    for u,v in sorted(sorted((u,v)) for u,v in G.edges()):
```

Comment:
Not in general, but in this case there is a relabeling of the nodes of the input graph to integer labels according to rcm ordering (see lines 230 and 344 from `current_flow_betweenness.py
`). So I think it's safe to assume the nodes can be sorted safely.